ophyd/scaler.py

```diff
@@ -5,7 +5,7 @@
 from .ophydobj import Kind
 from .signal import (EpicsSignal, EpicsSignalRO)
 from .device import Device
-from .device import (Component as C, DynamicDeviceComponent as DDC,
+from .device import (Component as Cpt, DynamicDeviceComponent as DDC,
                      FormattedComponent as FC)
```

Comment:
There was the `FCpt` usage somewhere else in this PR. Should it also be changed here?